### PR TITLE
OpenSubDiv: Build fix

### PR DESCRIPTION
**Pull request by extrowerk**
Needs review, i'm not sure if it is the right way and if it secondary-arch friendly.

Closes https://github.com/haikuports/haikuports/issues/2920

### CODE REVIEW

**Comment by extrowerk** [comment]
It builds fine on 64 bit.

**Comment by extrowerk** [comment]
@korli They are. Will test it...

File: media-libs/opensubdiv/opensubdiv-3.3.2.recipe [hunk 1]
```diff
@@ -83,8 +83,8 @@ BUILD()
 		GLEW_SUBDIR="$secondaryArchSuffix/"
 	fi
 
-	export GLEW_LOCATION="$portPackageLinksDir/glew${secondaryArchSuffix}_devel/develop"
-	export GLEW_INC_LOCATION="$portPackageLinksDir/glew${secondaryArchSuffix}_devel/develop/headers$UNITTEST_SUBDIR"
+	export GLEW_LOCATION="$portPackageLinksDir/lib~libglew/develop/"
+	export GLEW_INC_LOCATION="$portPackageLinksDir/lib~libglew/develop/headers/$GLEW_SUBDIR"
```
Comment (fbrosson) [inline media-libs/opensubdiv/opensubdiv-3.3.2.recipe]: Suggestion:
```diff
-	local GLEW_SUBDIR=
-	if [ -n "$secondaryArchSuffix" ]; then
-		GLEW_SUBDIR="$secondaryArchSuffix/"
-	fi
-
-	export GLEW_LOCATION="$portPackageLinksDir/lib~libglew/develop/"
+	export GLEW_LOCATION="$portPackageLinksDir/lib~libglew/$relativeDevelopDir/"
-	export GLEW_INC_LOCATION="$portPackageLinksDir/lib~libglew/develop/headers/$GLEW_SUBDIR"
+	export GLEW_INC_LOCATION="$portPackageLinksDir/lib~libglew/$relativeIncludeDir/"
```
Warning: I did not test this.

Same suggestion for version `3.3.3`.

BTW, unlike `3.3.3`, `3.3.2` is missing a `SOURCE_FILENAME`, so you might wish to add one:
```diff
 CHECKSUM_SHA256="d11a0f21bcf21156f9071ea49050ec6675b77776b9dba33b8e30b072d82a8b05"
+SOURCE_FILENAME="opensubdiv-$portVersion.tar.gz"
 SOURCE_DIR="OpenSubdiv-${portVersion//\./_}"
```

**Comment by Begasus** [comment]
To overcome the failing checks, git commit --amend (with no changes) should do the trick ...

**Pull request merged**